### PR TITLE
add sound feature

### PR DESCRIPTION
**Pull request by lfelipesv**
First implementation of the sound feature.

Sending the pull request for revision.

Is this the best way to implement it?

### CODE REVIEW

File: controllers/supervisor/supervisor.py [hunk 1]
```diff
@@ -461,6 +461,8 @@ def get_robot_touch_ball(self):
                 for id in range(constants.NUMBER_OF_ROBOTS):
                     if message[2 * id + team] == 1:
                         rc[team][id] = True
+                    if rc[team][id]:
+                        self.sound_speaker(0,1)
```
Comment (omichel) [inline controllers/supervisor/supervisor.py]:
```suggestion
                        self.sound_speaker(KICK_SOUND, 1)
```

**Review by omichel — APPROVED** [review]
Yes, that seems to be a good implementation of supervisor sounds. 👍
I just have a minor suggestion.

Please note however that these sounds will not be recorded if you create a movie from Webots. Movie creation from Webots doesn't support sound.

To record these sound along with a simulation movie, you could run Webots in fullscreen and use a third party software that record both the image and sound (I don't know these software, but they are many used mainly to create video game videos). Alternatively, you could record the movie from Webots, record the sound from a third party sound capture software (again, I have no special recommendation, but I am sure they exist) and use a video editing software to merge the silent video with the recorded sound track.

File: controllers/supervisor/supervisor.py (hunk 1 above)
Comment (lfelipesv) [reply controllers/supervisor/supervisor.py]: I probably forgot to change this one, hehe. Thanks!

**Review by omichel — APPROVED** [review]

**Comment by omichel** [comment]
Very good, I am merging this PR now.

**Pull request merged**